examples/seismic/acoustic/acoustic_example.py

```diff
@@ -52,18 +52,18 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 
     if preset == 'constant':
         # With  a new m as Constant
```

Comment:
new vp*

Reply:
you forgot these comments @mloubout . Can fix in the next PR